server/@types/forms/index.d.ts

```diff
@@ -57,7 +57,7 @@ declare module 'forms' {
       }>
       note?: string
     }>
-    action?: 'add-another-step' | 'submit-form'
+    action?: string
```

Comment:
We need to make this field a little loser now in respect of it's values. This is because there is an "add another step" button on each Goal on the form, and when one of these buttons is pressed it need to carry with it not only the action ('add-another-step'), but also the goal number that you want to add the step to.
The html form in the nunjucks template (below 👇 ) does this by suffixing the goal number on the button value:
```
value: "add-another-step|" + goalLoop.index0
```
Therefore if the form had 3 goals, there would be 3 "Add another step" buttons,  where their value attributes are:
1. `value="add-another-step|0`
2.`value="add-another-step|1`
3.`value="add-another-step|2`